dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets

```diff
@@ -8,13 +8,23 @@
   <!-- It should not display any schema warnings in error list window.            -->
   <!-- ========================================================================== -->
 
-  <!-- With .NET 5, the propery TargetPlatformIdentifier is not UAP but rather windows. This file expects UAP for all .NET versions, so adjust. -->
+  <!-- With .NET 5, the propery TargetPlatformIdentifier is not UAP but rather windows.
+       Likewise, for other console apps, TargetPlatformIdentifier is windows.
+       This file expects UAP for all .NET versions, so adjust. -->
```

Comment:
>This file expects UAP for all .NET versions, so adjust [](start = 7, length = 54)

then why match 'net5'?

Reply:
For the older versions, there is no need to match. It was already UAP.

Reply:
should we just always set it UAP?

Reply:
I think this is fine. It may even be preferred over setting it to UAP for all cases.